core/services/head_tracker.go

```diff
@@ -407,10 +411,8 @@ func (ht *HeadTracker) fetchAndSaveHead(ctx context.Context, n int64) (models.He
 func (ht *HeadTracker) subscribe() bool {
 	ht.sleeper.Reset()
 	for {
-		err := ht.unsubscribeFromHead()
-		if err != nil {
-			ht.logger.ErrorIf(err, "failed when unsubscribe from head")
-			return false
+		if err := ht.unsubscribeFromHead(); err != nil {
+			ht.logger.Error("failed when unsubscribe from head", err)
```

Comment:
did you mean to remove the `return false` here?

Reply:
did not, good catch there~